src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -384,41 +384,41 @@ osx_output_set_device_format(AudioDeviceID dev_id, const AudioStreamBasicDescrip
 	};
 
 	UInt32 property_size;
-	OSStatus err = AudioObjectGetPropertyDataSize(dev_id, &aopa, 0, NULL, &property_size);	
-	if (err != noErr) {	
-		throw FormatRuntimeError("Cannot get number of streams: %d\n", err);	
-	}	
+	OSStatus err = AudioObjectGetPropertyDataSize(dev_id, &aopa, 0, NULL, &property_size);
```

Comment:
Whitespace-only changes don't belong here. They just obfuscate the real changes and make review hard.

Reply:
Could you do a white space only commit to this file? The IDE tries to fix the formatting each time I open the file.

Reply:
Mark as resolved.